opm/common/utility/FileSystem.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+  Copyright 2019 Equinor ASA
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef OPM_FILESYSTEM_HPP
+#define OPM_FILESYSTEM_HPP
+
+
+#if __cplusplus__ >= 201703
```

Comment:
**Very** strictly speaking, I think you need to write
```C++
#if __cplusplus__ >= 201703L
```
here.

Reply:
fixed.